TGM/src/main/java/network/warzone/tgm/util/InventoryUtil.java

```diff
@@ -76,6 +84,11 @@ public static Color colorFromTime() {
         return Color.fromRGB(0);
     }
 
+    private static ItemStack getKillstreakItem(int killstreak) {
+        int amount = killstreak > 64 ? 1 : killstreak;
+        return ItemFactory.createItem(Material.IRON_SWORD, ChatColor.GREEN + "Killstreak: " + ChatColor.DARK_GREEN + killstreak, amount);
```

Comment:
Perhaps it'd be a good idea to make it display a diamond sword instead if the killstreak is over 64

Reply:
Done, however, it's not tested ingame. I also kept the amount of the item at 1 if the killstreak is above 64, because if it was stuck on 64, it would look stupid imo.